nodejs/week3/warmup/index.js

```diff
@@ -0,0 +1,130 @@
+const express = require("express");
+const app = express();
+
+//app.get("/", (req, res) => res.send("nodejs week3 homework"));
+
+app.get("/calculator/add", (req, res) => {
+  const firstParam = parseInt(req.query.firstParam);
+  const secondParam = req.query.secondParam;
+  let sum = firstParam;
+  if (Array.isArray(req.query.secondParam)) {
+    for (let i = 0; i < secondParam.length; i++) {
+      sum += parseInt(secondParam[i]);
+    }
+  } else {
+    sum += parseInt(secondParam);
+  }
```

Comment:
```suggestion
 const values = Object.values(req.query).flat()
 const sum = values.reduce((acc, val) => Number(acc) + Number(val))
```
It can be shortened, as for example, and similar for every operation: subtract, multiply, divide